Sources/SwiftCliCore/Errors.swift

```diff
@@ -3,8 +3,17 @@ import Foundation
 enum InputError: Error, Equatable {
     case emptyRow
     case invalidStitch(invalidStitch: String, rowLocation: Int? = nil)
-    case invalidRowWidth(invalidRowNumber: Int, expectedStitchCount: Int, actualCount: Int)
-    case invalidStitchNumber(rowNumber: Int? = nil, invalidStitch: String, validStitchType: String, invalidStitchNumber: String)
+    case invalidRowWidth(
+        invalidRowNumber: Int,
+        expectedStitchCount: Int,
+        actualCount: Int
+    )
+    case invalidStitchNumber(
+        rowNumber: Int? = nil,
```

Comment:
Does `rowNumber` need to be optional still?

Reply:
Because of how the error handling on this branch is set up, I can't fix this on another branch without doing a major refactor of what `NestedArrayBuilder` is responsible for, but I did fix this on  the `MultipleErrorArray` branch.